### DISCUSSION TITLE
SwiftUI Support?

### DISCUSSION THREAD

**Discussion by esnssr** [scenee/FloatingPanel#604] [Ideas]
Hey!
Love what you have created here. I was going through the docs and I noticed there isn't support for SwiftUI. I know it can be done, but I would very much love it there is a support for SwiftUI right away from the package. 
(Literally asking for a friend lol)

Thanks!

**Comment by scenee** [comment]
Thank you for your patience. I've finally released the SwiftUI API. I hope you would try it and let me know your feedback.
https://github.com/scenee/FloatingPanel/releases/tag/3.0.0
